### ISSUE TITLE
Regression: Rainbow Flashing Textures on Tomb Raider Trilogy (#9613)

### ISSUE THREAD

**Issue by SteveOfWarr** [RPCS3/rpcs3#9648]
## Quick summary
Since updating to v0.0.14-11619-22584fb2 I've gotten flashing rainbow coloured texture issues when playing the Tomb Raider Trilogy.

## Details
Previous version this game was working fine on was 'RPCS3 v0.0.14-11566-a692adef Alpha'

EDIT: Problem does not appear when you downgrade to RPCS3 Version: 0.0.14-11605-b96864c7 Alpha.

Broken Update Log:

[RPCS3.log](https://github.com/RPCS3/rpcs3/files/5862200/RPCS3.log)

Working Update Log:
[RPCS3.log](https://github.com/RPCS3/rpcs3/files/5862206/RPCS3.log)

Footage of issue from Discord post:
https://cdn.discordapp.com/attachments/277227681836302338/802875361209614386/rpcs3.exe_2021.01.24_-_12.14.12.02_1.mp4

#### 6. Please provide your system configuration:
* OS: Windows 10 Pro 20H2
* Ryzen 3700x
* RTX 2080
* 461.09
* Ram: 24GB

**Comment by AniLeo** [comment]
Which build introduced the regression? There are many builds between 11566 and 11619

**Comment by SteveOfWarr** [comment]
> Which build introduced the regression? There are many builds between 11566 and 11619

This one, the latest at the time of posting
 https://github.com/RPCS3/rpcs3/pull/9613

**Comment by SteveOfWarr** [comment]
Issues disappear when you downgrade to RPCS3 Version: 0.0.14-11605-b96864c7 Alpha.

**Comment by kd-11** [comment]
There's like 4 issues mentioned in the post. Which ones are caused by the last update?

**Comment by SteveOfWarr** [comment]
> There's like 4 issues mentioned in the post. Which ones are caused by the last update?

The Rainbow Texture one. For clarity, I've removed the others

**Comment by kd-11** [comment]
FTR, I figured out what to do for this issue, but it needs a lot of missing support infrastructure and testing - instead of simply disabling the passthrough feature on NVIDIA which is a simple temporary fix. This should be finished in the upcoming days.

**Comment by Megamouse** [comment]
NASCAR Unleashed regressed as well in the same PR:
The intro videos are bugged (not only the logos but also the racing afterwards)

![image](https://user-images.githubusercontent.com/23019877/107443231-bdb49080-6b38-11eb-88e3-06e7046bc072.png)

**Comment by DefaltBR** [comment]
The exact same issue started happening with The Last of Us after #9613... Only thing is, in TLoU it occurs less often, but it's there. It was fine before the DMA PR.

**Comment by Asinin3** [comment]
DMA Pull Request also broke Uncharted 1 unless "Write Depth Buffers" is enabled player shadows break, the graphics are blurry and parts of Drake's eyes are visible through his face model leading to:
![image](https://user-images.githubusercontent.com/15137035/107784545-81349f00-6d9b-11eb-8119-86cf23dfbcc1.png)
 
 [RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/5972577/RPCS3.log.gz)

**Comment by RainbowCookie32** [comment]
For Uncharted 1's issue, it also happens on Windows AMD drivers, but RADV is unaffected.

**Comment by kd-11** [comment]
Closing. Disabling asynchronous texture streaming correctly disables passthrough with NVIDIA. This issue is also completely solved by using asynchronous streaming for this particular game.

**Comment by Megamouse** [comment]
that being said, the NASCAR intro still flickers weirdly and I get some random glitches towards the end of the video.
And I can't really see a different with either Async streaming on or off

**Comment by Shoegzer** [comment]
Not sure if this is related but I"m still getting black flickering in Motorstorm even with async streaming off.  I thought this PR was supposed to clean remaining black flickering up since it was already improved from another PR but i could be wrong.  Using linux mint 20.1 with nvidia gtx 3070 with 460.38.

**Comment by kd-11** [comment]
The checks were implemented for windows only at this time. I'll extend it to support linux as well.
This will be handled under async streaming meta issue.
